cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-multiclusterhub-operator-pull-secret/externalsecret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+namespace: open-cluster-management-observability
+metadata:
+  name: open-cluster-management-observability-multiclusterhub-operator-pull-secret
+  namespace: open-cluster-management-observability
+spec:
+  refreshInterval: 15s
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  target:
+    name: multiclusterhub-operator-pull-secret
+  dataFrom:
+    - extract:
+        key: $ENV/$CLUSTER/open-cluster-management-observability-multiclusterhub-operator-pull-secret
```

Comment:
you need to patch this key `$ENV/$CLUSTER/...` in the overlay for nerc-ocp-prod

Reply:
Thanks @naved001 I have patched these values for nerc-ocp-prod in the latest commits.